filter_customs.py

```diff
@@ -123,7 +123,7 @@ def process_chapter_rec(chapter):
     """
 
     inp = sys.stdin.read()
-    with open("inp.json", "a") as ofile: ofile.write(inp)
+    #with open("inp.json", "a") as ofile: ofile.write(inp)
```

Comment:
Delete code instead of commenting it out.

Reply:
We might want to re-enable that in the future; I've thought I was done, then had to re-enable it before.